### PR TITLE
Add YAML file support

### PR DESCRIPTION
**Pull request by tbachman**
This adds support for YAML file formats in the
aci_asr_config.ini file.

Closes-issue: 456

### CODE REVIEW

File: networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py [hunk 1]
```diff
@@ -227,6 +227,53 @@ def test_transit_nets_cfg_invalid_file_format(self):
...
+                 cidr_exposed: 1.209.200.1/24
+                 segmentation_id: 1066
+            """
+        )
+        # thbachman: couldn't get assertRaises to work here,
+        # so used this construct instead
+        try:
+            # just accessing the member should trigger the exception
+            self.plugging_driver.transit_nets_cfg
+            self.assertTrue(True)
```
Comment (kentwu1) [inline networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py]: we can just remove this line.

File: networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py [hunk 2]
```diff
@@ -227,6 +227,53 @@ def test_transit_nets_cfg_invalid_file_format(self):
...
+            """
+        )
+        # thbachman: couldn't get assertRaises to work here,
+        # so used this construct instead
+        try:
+            # just accessing the member should trigger the exception
+            self.plugging_driver.transit_nets_cfg
+            self.assertTrue(False)
+        except aci_vlan.AciDriverConfigInvalidFileFormat:
+            self.assertTrue(True)
```
Comment (kentwu1) [inline networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py]: minor: u can just do pass() here I think.

File: networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py (hunk 1 above)
Comment (tbachman) [reply networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py]: will remove

File: networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py (hunk 2 above)
Comment (tbachman) [reply networking_cisco/tests/unit/cisco/device_manager/test_aci_vlan_trunking_driver.py]: will change to pass

**Comment by tbachman** [comment]
pushed update, addressing reviewer comments.

**Pull request merged**